docs/design/labelling.rst

```diff
@@ -0,0 +1,800 @@
+Guidelines for labelling the Mautic interface
+============================================
+
+Capitalization
+--------------
+
+Use sentence—case capitalization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Sentence-style capitalization is the preferred method for UI text elements due to its readability and efficiency. This style, which is predominantly lowercase, makes it easy for readers to distinguish between common nouns and proper nouns, and is generally considered the quickest Form to read.
+
+When using sentence-case capitalization, only capitalize the initial letter of the first word in the text and other words that require capitalization, such as proper nouns.
+
+For example, you would write labels in a Form as "First name" and "Email address." Apply this approach consistently across all UI text elements, except when the text is a product or service name, or trademarked.
+
+Examples:
+
+.. vale off
+
+- Analyse your lead generation chart with Marketing Insights.
+- Automate your marketing Campaigns.
+
+.. vale on
+
+Avoid title case capitalization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Title or headline case capitalization involves capitalizing the first letter of most words, excluding articles, conjunctions, or prepositions unless they're the first or last word in the sentence or phrase.
+
+However, this approach presents several challenges in practical implementation:
+
+- It requires all content creators within an organization to understand and consistently apply complex grammatical rules regarding word capitalization.
+- It often relies on subjective interpretations of what constitutes "important" or "special" words, which leads to inconsistencies if not clearly communicated or understood across the organization.
+- Moreover, title case can impede reading speed and comprehension, as it makes it more difficult for readers to distinguish between proper nouns and common nouns.
+
+These factors combined make title case less desirable in many user interface and content design scenarios, especially when prioritizing clarity and ease of reading.
+
+**Do:** automate your marketing Campaigns
+
+**Avoid:** automate your marketing Campaigns
```

Comment:
This is the same as the Do one - do you mean like this:

```suggestion
**Avoid:** Automate Your Marketing Campaigns
```

Reply:
I have to revert changes from everyone for this line, as it's making the docs provide the wrong example of capitalisation for the UI

Reply:
Done